packages/website/src/components/Playground.tsx

```diff
@@ -37,6 +37,9 @@ function Playground(): JSX.Element {
   const [visualEslintRc, setVisualEslintRc] = useState(false);
   const [visualTSConfig, setVisualTSConfig] = useState(false);
 
+  const [enableScrolling, setEnableScrolling] = useState<boolean>(true);
+  const [showTokens, setShowTokens] = useState<boolean>(false);
```

Comment:
do we want to save this in url / local storage?

Reply:
I think so, yes please!